static/_app/tools/tradecraft/index.html

```
@@ -0,0 +1,21 @@
+<meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no" />
+<title>Tradecraft</title>
+<link rel="shortcut icon" href="/boltlogo.png">
+<link rel="canonical" href="https://slope-game.github.io/tradecraft.html">
+<meta 
+     name='viewport' 
+     content='user-scalable=0' 
+/>
+<script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
+<script src='https://cdn1.kongregate.com/javascripts/kongregate_api.js'></script>
+<link href="style.css" rel="stylesheet">
+<script type="text/javascript" src=game/main.js></script>
```

HTMLHint (1.5.0)

[error] The value of attribute [ src ] must be in double quotes.

(attr-value-double-quotes)